src/main/java/frc/robot/subsystems/Transporter/Transporter.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.Transporter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Transporter extends SubsystemBase {
+    private final TalonSRX rightMotor = TransporterConstants.RIGHT_MOTOR;
+    private final TalonSRX leftMotor = TransporterConstants.LEFT_MOTOR;
+
+    void setVoltageOutput(double percentageMotorOutput) {
```

Comment:
is this the voltage output? also, call this setTagret<something>. you can never set the current value, only the wanted or target value